tests/orca_verifier.ts

```diff
@@ -0,0 +1,118 @@
+import * as anchor from '@coral-xyz/anchor';
+import { Program, Provider } from '@project-serum/anchor';
+import { TOKEN_PROGRAM_ID } from '@solana/spl-token';
+import { PublicKey } from '@solana/web3.js';
+import { DualAirdrop } from '../target/types/dual_airdrop';
+import { OrcaVerifier } from '../target/types/orca_verifier';
+import { createMint, createTokenAccount, mintToAccount } from './utils/utils';
+
+const crypto = require('crypto');
+
+describe('orca_verifier', () => {
+  anchor.setProvider(anchor.AnchorProvider.env());
+  const provider: Provider = anchor.AnchorProvider.env();
+
+  const airdropProgram = anchor.workspace.DualAirdrop as Program<DualAirdrop>;
+  const verifierProgram = anchor.workspace.OrcaVerifier as Program<OrcaVerifier>;
+
+  it('Orca Claim', async () => {
+    const verifierSeed = crypto.randomBytes(32);
+    const [orcaVerifierState, _orcaVerifierBump] = (
+      anchor.web3.PublicKey.findProgramAddressSync(
+        [verifierSeed],
+        verifierProgram.programId,
+      ));
+
+    const airdropAmount = new anchor.BN(1_000_000);
+    const mint = await createMint(provider, provider.publicKey);
+
+    const orcaSeed = crypto.randomBytes(32);
+    const [airdropState, _stateBump] = anchor.web3.PublicKey.findProgramAddressSync(
+      [orcaSeed],
+      airdropProgram.programId,
+    );
+    const [orcaVault, _bump] = anchor.web3.PublicKey.findProgramAddressSync(
+      [
+        Buffer.from(anchor.utils.bytes.utf8.encode('Vault')),
+        airdropState.toBuffer(),
+      ],
+      airdropProgram.programId,
+    );
+    const [verifierSignature, _signatureBump] = anchor.web3.PublicKey.findProgramAddressSync(
+      [airdropState.toBuffer()],
+      verifierProgram.programId,
+    );
+
+    await airdropProgram.methods.configure(
+      orcaSeed,
+    )
+      .accounts({
+        payer: provider.publicKey,
+        state: airdropState,
+        vault: orcaVault,
+        mint,
+        verifierSignature,
+        tokenProgram: TOKEN_PROGRAM_ID,
+        systemProgram: anchor.web3.SystemProgram.programId,
+        rent: anchor.web3.SYSVAR_RENT_PUBKEY,
+      })
+      .rpc({ skipPreflight: true });
+
+    await mintToAccount(provider, mint, orcaVault, airdropAmount, provider.publicKey);
+
+    const rewardIndex = 0;
+    const position = new PublicKey('HA1hk5mneoFcdwe9wC33eaofSeo9yVkXPsAonLm7jUCR');
```

Comment:
Any significance to these?

Reply:
just need accounts of the correct type. easiest way to do that is make ones on mainnet and clone them to localnet